repl/evaluator_test.go

```diff
@@ -86,7 +97,24 @@ func TestLetError(t *testing.T) {
 		t.Fatalf("NewEvaluator() failed with: %v", err)
 	}
 
-	err = eval.AddLetVar("y", "z + 1")
+	err = eval.AddLetVar("y", "z + 1", nil)
+	if err == nil {
+		t.Errorf("eval.AddLetVar() got non-error, wanted error")
```

Comment:
Note the error here (and below) is formatted via `Errorf`, but does not provide any params. Did you mean to provide some additional state?

Reply:
done